packages/side-nav/theme/lumo/vaadin-side-nav-item-styles.js

```diff
@@ -88,6 +92,10 @@ export const sideNavItemStyles = css`
     color: var(--lumo-contrast-60pct);
   }
 
+  :host([disabled]) slot[name='prefix']::slotted(:is(vaadin-icon, [class*='icon'])) {
```

Comment:
Just to make sure, are we not using the disabled color for the Material theme icons?

Reply:
Good catch, thanks. I missed to add these styles.